Console.WriteLine

Console.WriteLine = SHOULD BE **System.Console.WriteLine** 

### error while run example code.

throwing error while run this page as below.
program.cs(30,9): error CS0103: The name 'Console' does not exist in the current context
program.cs(34,9): error CS0103: The name 'Console' does not exist in the current context
program.cs(36,9): error CS0103: The name 'Console' does not exist in the current context


---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 63989d15-db4e-9873-2cf7-77634496fe4e
* Version Independent ID: cd6c869c-fcc9-eb27-4ec0-94cb1e961a88
* Content: [Classes](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/types/classes#code-try-6)
* Content Source: [docs/csharp/fundamentals/types/classes.md](https://github.com/dotnet/docs/blob/main/docs/csharp/fundamentals/types/classes.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

--- a/docs/csharp/fundamentals/types/snippets/classes/Program.cs
+++ b/docs/csharp/fundamentals/types/snippets/classes/Program.cs
@@ -1,4 +1,6 @@
-﻿public class Person
+using system;
+
+public class Person
 {
     // Constructor that takes no arguments:
     public Person()

--- a/docs/csharp/fundamentals/types/snippets/classes/Program.cs
+++ b/docs/csharp/fundamentals/types/snippets/classes/Program.cs
@@ -1,4 +1,4 @@
-using system;
+using System;
 
 public class Person
 {

